kong/plugins/grpc-gateway/deco.lua

```diff
@@ -295,5 +355,4 @@ function deco:get_raw_downstream_body()
   return self.downstream_body
 end
 
-
-return deco
+return deco
```

Comment:
Not introduced by your PR but we need a new line at the end of the file.

Reply:
https://github.com/Kong/kong/pull/8297/commits/650b43abfff8374a638dc8ae516da93c0fb964ca

Reply:
I mean a newline after `return deco`.
```suggestion
return deco

```